docs/content/contribute/devlunch.md

```diff
@@ -3,9 +3,17 @@ title: Lunch with the Devs
 description: Contact Us
 ---
 
-{{% usa-intro %}}To increase communication with the OSCAL community, the NIST OSCAL team is hosting a one-hour teleconference every two weeks starting on Thursday, December 5th 2019 @ noon EST.{{% /usa-intro %}}
+{{% usa-intro %}}To increase communication with the OSCAL community, the NIST OSCAL team is hosting a one-hour teleconference every two weeks starting on Thursday, December 5th 2019 @ noon EST.{{% /usa-intro %}}Global
```

Comment:
This change probably doesn't belong in this PR

Reply:
Agreed. We need to do a better job of keeping PRs more focused on set of related changes.